Essentials/src/messages.properties

```diff
@@ -490,6 +490,7 @@ weatherStorm=\u00a76You set the weather to \u00a7cstorm\u00a76 in\u00a7c {0}\u00
 weatherStormFor=\u00a76You set the weather to \u00a7cstorm\u00a76 in\u00a7c {0} \u00a76for {1} seconds.
 weatherSun=\u00a76You set the weather to \u00a7csun\u00a76 in\u00a7c {0}\u00a76.
 weatherSunFor=\u00a76You set the weather to \u00a7csun\u00a76 in\u00a7c {0} \u00a76for {1} seconds.
+whoisProxyAddress=\u00a76 - Proxy Address\:\u00a7r {0}
```

Comment:
Needs to be put in every message file. Just put the english version in and someone can translate it later.